### DISCUSSION TITLE
Welcome to Discussions about IONOS Cloud Terraform Provider !

### DISCUSSION THREAD

**Discussion by oana-ungureanu-ionos** [ionos-cloud/terraform-provider-ionoscloud#245] [General]
We consider collaboration the pillar for building a stronger community so we welcome you to use **Discussions** as a place to:

- [ ]  Start conversations. 
- [ ]  Ask or answer questions.
- [ ]  Share ideas.
- [ ]  Post updates and announcements.
- [ ]  Provide feedback.
- [ ]  Engage with maintainers and other community members.

To get started, we encourage you to write a short introduction about yourself and how this community can serve your project.

**Comment by mrndev** [comment]
Hi - the documentation README could have a line or paragraph for the normal users saying that building and installing the provider yourself is not necessary and that one just needs to include the terraform provider into the tf file and run "terraform init" to get the provider installed.

**Reply by mrndev** [reply]
A customer of mine was wondering this (they started to build the provider) and I remember also thinking about whether I need to build the provider myself before I just tried if it works with terraform out of the box (which it did :)

**Reply by cristiGuranIonos** [reply]
Thanks, we will change the docs to be more clear on the point that you don't need to build the provider in order to use it.
